waves/project.py

```diff
@@ -1592,19 +1594,25 @@ def energy_losses(
             by="turbine",
             units="kw",
         )
+
         production = self.energy_production(  # type: ignore
             "month-year",
             by="turbine",
             units="kw",
-            with_losses=with_losses,
-            loss_ratio=loss_ratio,
         )[self.wombat.metrics.turbine_id]
+
         losses = potential - production
 
+        # Compute total loss ratio
+        total_loss_ratio = self.total_loss_ratio(environmental_loss_ratio = environmental_loss_ratio)
```

Comment:
It looks like this is unused, but it should be used to compute `losses = potential * (1 - total_loss_ratio)`.

Reply:
I am guessing you mean losses = potential * total_loss_ratio